### ISSUE TITLE
Task - Image matrix (Edit mode) - moving images from cell to cell

### ISSUE THREAD

**Issue by proceps** [SpeciesFileGroup/taxonworks#3111]
he image matrix, when you have a cell with several depictions, and you move one of them to another cell, several depiction just disappear. and you have to reload the page to see depictions (those still stays in the original cell). 
The moving functionality is working. You just need to reload the page after moving each image

**Comment by jlpereira** [comment]
This could be replicated in local? or just happen in production?

**Comment by proceps** [comment]
I replicated it on local. Added 3 images to OTU. Added OTU to an image matrix. When I move the first images from OTU to a descriptor, some of the images in the OTU cell disappear. But they are still there and visible when the page is reloaded (this happened to me in about 50% of the experiments).

**Comment by jlpereira** [comment]
I can't replicate this in local.

I added some images to an OTU, then I opened Image Matrix and moved those OTU depictions to an observation cell. Could you check if you're getting a console error? Or the exactly steps to replicate it? A screenshot of how the interface looks before the bug and after could help too

**Comment by proceps** [comment]
I just did it in sanstorm and local:
1. I already have an image matrix with several descriptors.
2. I went to OTU page and used radial annotator to add 6 images to OTU.
3. On the same page, I used OTU radial to add this OTU to the matrix.
4. Switched matrix to Edit mode
5. I have 6 depictions in the first column, when I move the first (top) depiction to another cell. I have only 4 depictions left in the first column. I move the top depiction again, and I have 2 depictions left in the first column. I move one more depiction, and non is left in the first column. If I reload the page, 3 depictions are in the first column. 

Moving images around works perfect, all images are moved and reassigned properly. I do not see any error messages in the consul.
